manifests/b/Blizzard/BattleNet/1.18.10.3141/Blizzard.BattleNet.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Blizzard.BattleNet
+PackageVersion: 1.18.10.3141
+InstallerType: exe
+InstallerSwitches:
+  Silent: --lang=enUS --installpath="C:\Program Files (x86)\Battle.net"
+  SilentWithProgress: --lang=enUS --installpath="C:\Program Files (x86)\Battle.net"
+Installers:
+- InstallerUrl: https://downloader.battle.net/download/getInstaller?os=win&installer=Battle.net-Setup.exe
+  Architecture: x86
+  InstallerSha256: 002F33FEE7B8A159058368B7E93E492931C4CA72E90660BDB2691BCD62FEDD3C

```

```suggestion
- InstallerUrl: https://downloader.battle.net/download/getInstaller?os=win&installer=Battle.net-Setup.exe
  Architecture: x86
  InstallerSha256: 002F33FEE7B8A159058368B7E93E492931C4CA72E90660BDB2691BCD62FEDD3C
  InstallerLocale: en-US
  InstallerSwitches:
    Custom: --lang=enUS
    InstallLocation: --installpath="<INSTALLPATH>"
  InstallLocationRequired: true
```
The install path should not be statically set, instead, the `InstallLocation` switch should be used. Additionally, because the locale is being set by a switch, the `InstallerLocale` needs to be set also. This way the other locales can be added by adding a different installer node